packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeLink/MT5TradeLink.tsx

```diff
@@ -54,24 +55,28 @@ const MT5TradeLink: FC<TMT5TradeLinkProps> = ({ app = 'linux', platform, webtrad
             </div>
             {(platform === mt5Platform || app === ctraderPlatform) && (
                 <Button
+                    className='border-opacity-black-400 rounded-200 px-800'
+                    colorStyle='black'
                     onClick={() => window.open(app === 'web' ? webtraderUrl : content.link)}
                     size='sm'
-                    variant='outlined'
+                    variant='secondary'
                 >
                     {content.text}
                 </Button>
             )}
             {platform !== mt5Platform && app !== ctraderPlatform && (
                 <Button
                     className='flex justify-center items-center border-none rounded-md cursor-pointer bg-system-dark-primary-background p-400 gap-400'
-                    colorStyle='black'
+                    colorStyle='white'
                     onClick={onClickWebTerminal}
                     variant='secondary'
                 >
-                    {PlatformToLabelIconMapper[platform ?? dxtradePlatform]}
-                    <Text bold colorStyle='white' size='sm'>
-                        Web terminal
-                    </Text>
+                    <span className='flex justify-center items-center gap-400'>
+                        {PlatformToLabelIconMapper[platform ?? dxtradePlatform]}
+                        <Text bold className='text-[white]' colorStyle='white' size='sm'>
```

Comment:
Why do we need and extra className for this?
If you want pure white:
```suggestion
                        <Text bold className='text-system-light-primary-background' size='sm'>
```